### DISCUSSION TITLE
Throwing away stealth

### DISCUSSION THREAD

**Discussion by tyson-dev** [isid0re/SoloLeveling#312] [General]
Seen now both hammer and cold sorc throw away stealth after making it. Looks like they don't like it if they can't equip it yet :D (seen sorc @ level 16 drop at stash, console sayd runeword kept) anyone can confirm? From Nightmare on Smoke is fine, saw my hammer do his with help of NM Hellforge LUM.

Edit: She just made a new one @level17 and equipped correctly ofc, so bot wasting tal+eth when <level17, probably intended to keep rw making simple?

**Comment by ForestRock22** [comment]
I think I had something similar happen a couple times. My necro droped Heart of the oak after making it.

**Comment by tyson-dev** [comment]
So i guess what happens is a check if there are EQUIPPABLE! items of a higher tier, and if not it does the "clear trash" thing to keep inventory and stash manageable. Basically cannot equip>throw away i guess. not sure about the hoto though, any requirements not met for that thing Forestrock22, hoto/Vex is level req 55, maybe made into scourge which has level req 57 or something similar? Can u actually make a Hoto before hitting level 55?

Edit: ofc you can make HOTO earlier :D Same as with Stealth! So This probably really starts to hurt when you throw away botds, enigmas and Infys harhar

**Reply by ForestRock22** [reply]
yeah needs higher dex then he had
